inc/TRestWimpUtils.h

```diff
@@ -31,7 +31,7 @@ namespace TRestWimpUtils {
 /// Physics constants
 constexpr double GEV_PER_UMA = 0.93149432;
 constexpr double HC_KEV_FM = 197327.053;
-constexpr double LIGHT_SPEED = 300000.0;  // km/s
```

Comment:
It would be better to use directly `TRestPhysics` where physical constants are defined

https://github.com/rest-for-physics/framework/blob/master/source/framework/tools/inc/TRestPhysics.h

Reply:
> It would be better to use directly `TRestPhysics` where physical constants are defined
> 
> https://github.com/rest-for-physics/framework/blob/master/source/framework/tools/inc/TRestPhysics.h

I think we should have a dedicated discussion about this since REST has different units that the ones that are used in wimplib, I will create an issue about this.

Reply:
Which different units? At least the common ones should be common.

Reply:
For instance we are using `km/s` for the speed, because it is reasonable for the WIMP velocity distribution, so we use `km/s` for the light speed.

I think this should be discussed in the dedicated issue and should be implemented in a different PR.